example/example-node.ts

```diff
@@ -21,7 +21,7 @@ async function example() {
     });
 
     console.log('Getting big map key...');
-    await tezos.contract.at('KT1AbzoXYgGXjCD3Msi3spuqa5r5MP3rkvM9').then(async contract => {
+    await tezos.contract.at('KT1VBn1rTZ4FLz43hC9xugn56K9mSM2DQQXJ').then(async contract => {
```

Comment:
Not the right contract

Reply:
changed to KT1AwUJp6ozYtzhpf5wVXZPQSFxb64JFcVvi

Reply:
Verified the example works with new contract